static/js/components/UserChip_test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { assert } from 'chai';
+
+import { USER_PROFILE_RESPONSE } from '../constants';
+import R from 'ramda';
+import UserChip from './UserChip';
+import { getPreferredName } from '../util/util';
+import ProfileImage from './ProfileImage';
+
+describe('UserChip', () => {
+  let clone;
+
+  beforeEach(() => {
+    clone = R.clone(USER_PROFILE_RESPONSE);
```

Comment:
For a lot of the redux stuff we end up modifying fields deep in an object, which led us to use the lodash `cloneDeep` method a lot, I think this is where the clone name originated.